Core/Services/StitchCoreRemoteMongoDBService/Tests/StitchCoreRemoteMongoDBServiceTests/CoreRemoteMongoClientUnitTests.swift

```diff
@@ -1,8 +1,10 @@
 import XCTest
 import Foundation
+import MongoMobile
 @testable import StitchCoreRemoteMongoDBService
 
-final class CoreRemoteMongoClientUnitTests: XCTestCase {
+final class CoreRemoteMongoClientUnitTests: XCMongoMobileTestCase {
```

Comment:
[q] this seems more a mongo mobile change not a sync hange?

Reply:
We were doing this improperly before. Unfortunately, MongoMobile is setup in a way that it can only be initialized once and torndown once. So we need to be very careful about where we do those operations, especially with test bundles.